### PR TITLE
Update to CODEOWNES scope

### PR DESCRIPTION
**Pull request by devindford**
### What does this PR do? What is the motivation?

- Updates the codeowner file to scope specific paths to webops platform
- Removes the dual codeowners from website build and configuration files
- Moves the webops platform section under the top docs/corpweb section (making things easier to navigate/find)
### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

After incident web-incident-30 we discovered that the websites team had some blindspots as well as missing approvals from pull requests that altered the way build scripts run. Due to github limitations, if there are 2 codeowners for a file, it will accept an either/or approval instead of requiring both to merge. This is a first step in a larger remediation around how approvals are required.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,16 +1,63 @@
...
+.github/workflows/check_cache_value.yml                       @DataDog/documentation
+.github/workflows/code-freeze.yml                             @DataDog/documentation
+.github/workflows/gif_check.yml                               @DataDog/documentation
+.github/workflows/preview_link.yml                            @DataDog/documentation
+.github/workflows/vale_linter.yml                             @DataDog/documentation
+local/bin/py/check_cache_values.py                            @DataDog/documentation
+local/bin/py/integration-finder.py                            @DataDog/documentation
+local/bin/py/missing_metrics.py                               @DataDog/documentation
+local/bin/py/preview_links.py                                 @DataDog/documentation
+local/bin/py/missing_metrics.py                               @DataDog/documentation
```
Comment (hestonhoffman) [inline .github/CODEOWNERS:57]:
```suggestion
```
Dupe

**Review by hestonhoffman — APPROVED** [review]
LGTM, one small edit

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,16 +1,62 @@
...
+local/bin/py/check_cache_values.py                            @DataDog/documentation
+local/bin/py/integration-finder.py                            @DataDog/documentation
+local/bin/py/missing_metrics.py                               @DataDog/documentation
+local/bin/py/preview_links.py                                 @DataDog/documentation
+local/bin/py/preview-links-template.mako                      @DataDog/documentation
+local/bin/py/build/configurations/pull_config_preview.yaml    @DataDog/WebOps-Platform @DataDog/documentation
+local/bin/py/build/configurations/pull_config.yaml            @DataDog/WebOps-Platform @DataDog/documentation
 
 # SDK versions
 .github/workflows/bump_versions.yml   @DataDog/api-clients @DataDog/documentation
```
Comment (bgdeutsch) [inline .github/CODEOWNERS:62]:
```suggestion
.github/workflows/bump_versions.yml   @DataDog/WebOps-Platform
/data/sdk_versions.json @DataDog/WebOps-Platform @DataDog/documentation
```

Comment (bgdeutsch) [inline .github/CODEOWNERS:62]:
```suggestion
.github/workflows/bump_versions.yml   @DataDog/WebOps-Platform
```

**Review by bgdeutsch — APPROVED** [review]

**Pull request merged**